core/templates/domain/admin/admin-backend-api.service.spec.ts

```diff
@@ -26,7 +26,7 @@ import { ComputationDataObjectFactory } from
   'domain/admin/computation-data-object.factory';
 import { JobDataObjectFactory } from
   'domain/admin/job-data-object.factory';
-import { JobStausSummaryObjectFactory } from
+import { JobStatusSummaryObjectFactory } from
```

Comment:
@brianrodri, Thanks for fixing this!

/cc @nishantwrp, FYI, we had made this mistake!